tests/helpers.py

```diff
@@ -609,6 +609,39 @@ def create_multiple_pvcs(
     ]
 
 
+def create_pvc_clone(
+        sc_name, parent_pvc_name, size=None, pvc_name=None, do_reload=True, volume_mode=None
+):
+    """
+    Create a cloned pvc from existing pvc
+    Args:
+        sc_name (str): The name of the storageclass
```

Comment:
Can the storage class of cloned PVC be different from the storage class of parent PVC ?